interchain-exchange/tutorial/09-walkthrough.md

```diff
@@ -29,9 +29,27 @@ The ports for `Mars` are default ports, while for `Venus` the port details can b
 
 ```bash
 # relayer configuration
-starport relayer configure -a --source-rpc "http://0.0.0.0:26657" --source-faucet "http://0.0.0.0:4500" --source-port "ibcdex" --source-version "ibcdex-1" --target-rpc "http://0.0.0.0:26659" --target-faucet "http://0.0.0.0:4501" --target-port "ibcdex" --target-version "ibcdex-1"
+starport relayer configure -a \
+--source-rpc "http://0.0.0.0:26657" \
+--source-faucet "http://0.0.0.0:4500" \
+--source-port "ibcdex" \
+--source-version "ibcdex-1" \
+--target-rpc "http://0.0.0.0:26659" \
+--target-faucet "http://0.0.0.0:4501" \
+--target-port "ibcdex" \
+--target-version "ibcdex-1"
 ```
 
+when prompted enter `0.0000025token` as source and target gas price:
```

Comment:
I may have been a bit ambiguous in my previous message.  The source and target gas price options don't exist with the current released version of Starport therefore this prompt will not show up

Reply:
Add these changes here: https://github.com/cosmos/sdk-tutorials/pull/600